internal/uvm/virtual_device.go

```diff
@@ -15,54 +15,106 @@ import (
 // belongs to.
```

Comment:
Can we update this comment now that VPCIDevice holds more than just an ID and a handle to the UVM.